### ISSUE TITLE
Missing phantomjs 2.0.0 for linux

### ISSUE THREAD

**Issue by soujiro32167** [klieber/phantomjs-maven-plugin#35]
Using your `2.0-beta-2`, I am successfully running tests on windows and mac. On linux, however, the following exception is thrown (see below). It looks like a linux version of phantomjs 2.0.0 is missing, which I do not see here here: https://repo1.maven.org/maven2/com/github/klieber/phantomjs/2.0.0/

```
[INFO] jetty-8.1.14.v20131031
[INFO] Started SelectChannelConnector@0.0.0.0:42304
[INFO] Executing Jasmine Specs
[INFO] Downloading: http://wd-artifactory.autodesk.com/artifactory/repo/com/github/klieber/phantomjs/2.0.0/phantomjs-2.0.0-linux-x86_64.tar.bz2
[INFO] Downloading: https://repo.maven.apache.org/maven2/com/github/klieber/phantomjs/2.0.0/phantomjs-2.0.0-linux-x86_64.tar.bz2
[ERROR] Unable to locate phantomjs binary
com.github.klieber.phantomjs.install.InstallationException: Unable to install phantomjs.
	at com.github.klieber.phantomjs.install.PhantomJsInstaller.install(PhantomJsInstaller.java:55)
	at com.github.klieber.phantomjs.locate.ArchiveLocator.locate(ArchiveLocator.java:45)
	at com.github.klieber.phantomjs.locate.CompositeLocator.locate(CompositeLocator.java:39)
	at com.github.klieber.phantomjs.locate.PhantomJsLocator.locate(PhantomJsLocator.java:54)
	at com.github.searls.jasmine.driver.WebDriverFactory.createPhantomJsWebDriver(WebDriverFactory.java:145)
	at com.github.searls.jasmine.driver.WebDriverFactory.createWebDriver(WebDriverFactory.java:68)
	at com.github.searls.jasmine.mojo.TestMojo.createDriver(TestMojo.java:258)
	at com.github.searls.jasmine.mojo.TestMojo.executeSpecs(TestMojo.java:233)
	at com.github.searls.jasmine.mojo.TestMojo.run(TestMojo.java:202)
	at com.github.searls.jasmine.mojo.AbstractJasmineMojo.execute(AbstractJasmineMojo.java:396)
	at com.github.searls.jasmine.mojo.TestMojo.execute(TestMojo.java:189)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:132)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:120)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:355)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:155)
	at org.jvnet.hudson.maven3.launcher.Maven32Launcher.main(Maven32Launcher.java:132)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchStandard(Launcher.java:330)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:238)
	at jenkins.maven3.agent.Maven32Main.launch(Maven32Main.java:181)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at hudson.maven.Maven3Builder.call(Maven3Builder.java:136)
	at hudson.maven.Maven3Builder.call(Maven3Builder.java:71)
	at hudson.remoting.UserRequest.perform(UserRequest.java:121)
	at hudson.remoting.UserRequest.perform(UserRequest.java:49)
	at hudson.remoting.Request$2.run(Request.java:324)
	at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.github.klieber.phantomjs.download.DownloadException: Unable to resolve artifact.
	at com.github.klieber.phantomjs.download.RepositoryDownloader.download(RepositoryDownloader.java:64)
	at com.github.klieber.phantomjs.install.PhantomJsInstaller.install(PhantomJsInstaller.java:52)
	... 42 more
Caused by: org.eclipse.aether.resolution.ArtifactResolutionException: Could not find artifact com.github.klieber:phantomjs:tar.bz2:linux-x86_64:2.0.0 in autodesk-releases (http://wd-artifactory.autodesk.com/artifactory/repo)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:444)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifacts(DefaultArtifactResolver.java:246)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifact(DefaultArtifactResolver.java:223)
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveArtifact(DefaultRepositorySystem.java:294)
	at com.github.klieber.phantomjs.download.RepositoryDownloader.download(RepositoryDownloader.java:58)
	... 43 more
Caused by: org.eclipse.aether.transfer.ArtifactNotFoundException: Could not find artifact com.github.klieber:phantomjs:tar.bz2:linux-x86_64:2.0.0 in autodesk-releases (http://wd-artifactory.autodesk.com/artifactory/repo)
	at org.eclipse.aether.connector.basic.ArtifactTransportListener.transferFailed(ArtifactTransportListener.java:39)
	at org.eclipse.aether.connector.basic.BasicRepositoryConnector$TaskRunner.run(BasicRepositoryConnector.java:355)
	at org.eclipse.aether.util.concurrency.RunnableErrorForwarder$1.run(RunnableErrorForwarder.java:67)
	at org.eclipse.aether.connector.basic.BasicRepositoryConnector$DirectExecutor.execute(BasicRepositoryConnector.java:581)
	at org.eclipse.aether.connector.basic.BasicRepositoryConnector.get(BasicRepositoryConnector.java:249)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.performDownloads(DefaultArtifactResolver.java:520)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:421)
	... 47 more
```

**Comment by isuftin** [comment]
We are having the same issue on our shop. We've reverted back to 1.9.8 for the time being.

**Comment by soujiro32167** [comment]
Same here - 1.9.7

**Comment by larkin** [comment]
Running into the same issue, unfortunately, can't step back to version 1.9.8 as it isn't fully ES5 compliant where as 2.0 is compliant.

Unfortunately, looks like this is not @klieber 's issue, as phantomjs doesn't yet have a working linux binary for 2.0.0:  https://github.com/ariya/phantomjs/issues/12948

Will look into an alternate method for installing phantomjs on linux.

**Comment by soujiro32167** [comment]
True, that. In the end, I removed version sensitivity at the plugin level.
Adding the installed phantomjs to the path prevented, looking in the ariya
repo

**Comment by klieber** [comment]
Closing this since a Linux version of 2.0.0 was never released and now 2.1.1 is out there.
